src/responses.rs

```diff
@@ -16,6 +18,26 @@ pub struct OutpackError {
     pub detail: String,
 }
 
+impl OutpackError {
+    pub fn new(e: io::Error) -> OutpackError {
+        OutpackError {
+            error: e.kind().to_string(),
+            detail: e.to_string()
+        }
+    }
+}
+
+impl<'r> Responder<'r, 'static>  for OutpackError {
```

Comment:
ok, what is going on with all these apostrophes here, this looks very odd

Reply:
these are lifetimes, but don't ask me why this needs annotating or whether `'static` is the best choice here :see_no_evil: https://doc.rust-lang.org/book/ch10-03-lifetime-syntax.html